index.js

```diff
@@ -22,14 +22,14 @@ module.exports = {
   },
 
   treeForVendor(rawVendorTree) {
-    let babelAddon = this.addons.find(addon => addon.name === 'ember-cli-babel');
+    var babelAddon = this.addons.find(addon => addon.name === 'ember-cli-babel');
```

Comment:
We should add `'use strict'` to the top (instead of `var`)

Reply:
👍